index.html

```diff
@@ -32,5 +35,182 @@ <h3>LET'S CONNECT</h3>
+                </p>
+                <div class="media_contact">
+                    <h7 class="let">LET'S CONNECT</h7>
+                    <ul class="contacts">
+                        <li><a href="#"><img class="icons" src="Vector.svg"></a></li>
+                        <li><a href="#"><img class="icons" src="linkedin.svg"></a></li>
+                        <li><a href="#"><img class="icons" src="vector1.svg" alt=""></a></li>
+                        <li><a href="#"><img class="icons" src="vector2.svg" alt=""></a></li>
+                        <li><a href="#"><img class="icons" src="vector3.svg" alt=""></a></li>
+                    </ul>
+                </div>
+                <button class="resume button">Get my resume</button>
+            </div>
+            <div class="skill_list">
+                <div class="pgrm_languages">
+                    <div class="lang_head">
+                        <img class="lang_head_icon" src="downarrow.svg">
+                        <h6>Languages</h6>
+                    </div>
+                    <ul class="languange_list">
+                        <li class="lang_item">
+                            <p class="langs">Javascript</p>
+                            <img class="langs_img" id="js" src="js.svg">
+                        </li>
+                        <li class="lang_item">
+                            <p class="langs">HTML</p>
+                            <img class="langs_img" id="ht" src="html.svg">
+                        </li>
+                        <li class="lang_item">
+                            <p class="langs">CSS</p>
+                            <img class="langs_img" id="cs" src="css.svg">
+                        </li>
+                    </ul>
+                    <div class="containers">
+                        <span class="text">Frameworks</span>
+                        <img src="sidearrow.svg" alt="Your Image" class="image">
+                    </div>
+                    <img class="line" src="line.svg">
+                    <div class="containers">
+                        <span class="text">Skills</span>
+                        <img src="sidearrow.svg" alt="Your Image" class="image">
+                    </div>
+                    <img class="line" src="line.svg">
+                </div>
+            </div>
+        </section>
+
+
+        <section class="contact-form" id="cont-form">
+            <div >
+                <div class="contain">
+                    <h1>Contact Me</h1>
+                    <p>
+                        If you have an application you are interested in developing, a feature that you need built,
+                        or a project that needs coding, I'd love to help with it!
+                    </p>
+                    <form id="fs-frm" class="fs" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] The link you have provided is incorrect. please kindly follow the directions in this [link](https://formspree.io/html) so you can setup your formspree account and attach your correct id properly so that users can reach out to you properly :+1: